src/vma/dev/ring.h

```diff
@@ -81,23 +81,23 @@ typedef enum {
 /* udp uc key, only by destination port as we already know the rest */
 typedef struct __attribute__((packed)) flow_spec_udp_uc_key_t {
 	in_port_t 	dst_port;
-	flow_spec_udp_uc_key_t(): dst_port(INPORT_ANY){}//Default constructor
-	flow_spec_udp_uc_key_t(in_port_t d_port): dst_port(d_port){}//Constructor
+	flow_spec_udp_uc_key_t(){ memset(this, 0, sizeof(*this)); }//Default constructor
```

Comment:
Formatting comment: please align spaces for all functions....   `_key_t() { memse...(); }`
some have space and some are missing the extra space.